services/configuration/src/transactions.rs

```diff
@@ -93,7 +93,12 @@ fn enough_votes_to_commit(snapshot: &Snapshot, cfg_hash: &Hash) -> bool {
     let schema = Schema::new(snapshot);
     let votes = schema.votes_by_config_hash(cfg_hash);
     let votes_count = votes.iter().filter(|vote| vote.is_some()).count();
-    votes_count >= State::byzantine_majority_count(actual_config.validator_keys.len())
+
+    votes_count >=
```

Comment:
I suggest to move `match` into separate variable, just to fix weird formatting and make code more readable.
```rust
let majority_count = actual_config.majority_count {
    Some(majority_count) => majority_count as usize,
    _ => State::byzantine_majority_count(actual_config.validator_keys.len()),
};
votes_count >= majority_count
```

Reply:
bleh... rustfmt made me do it